app/models/article.rb

```diff
@@ -364,7 +364,7 @@ def self.unique_url_error
         :created_at
       end
 
-    order(column => dir.to_sym)
+    order("#{column} #{dir}")
```

Comment:
Hello @Anshuman71, thank you for your PR.

I will defer to other reviewers, but your change on this line seems less readable to me. I would remove the reassignment on line `355` and then on line `367`: `order(column: dir.to_sym)`

What do you think?

Reply:
@msarit  thanks for the feedback, I don't know Ruby at all I just looked up the correct answer. My goal was to highlight one possible solution.